### ISSUE TITLE
(URGENT) Duplicated "Terms and definitions" boilerplate

### ISSUE THREAD

**Issue by ronaldtse** [metanorma/metanorma-iso#497]
Source: (https://github.com/metanorma/iso-10303-2)

```adoc
== Terms, definitions and abbreviated terms

=== Terms and definitions

==== General

include::030-general.adoc[leveloffset=+2]

==== From STEPmod

include::031-generated-terms.adoc[leveloffset=+2]

=== Abbreviated terms

include::03x-abbrev.adoc[]

```

<img width="683" alt="Screenshot 2021-04-27 at 11 05 14 PM" src="https://user-images.githubusercontent.com/11865/116264635-08938b00-a7ad-11eb-80b2-88fed3842363.png">

<img width="676" alt="Screenshot 2021-04-27 at 11 05 27 PM" src="https://user-images.githubusercontent.com/11865/116264667-10ebc600-a7ad-11eb-9547-adc9f41f3605.png">

**Comment by ronaldtse** [comment]
The proper boilerplate in this case should apply to 3.1, not 3.1.1.

**Comment by opoudjis** [comment]
Standoc puts the boilerplate after each clause identified with `//sections/terms | //sections/clause[descendant::terms]`, which would put it in each of 3.1, 3.1.1, 3.1.2.

Iso overrides this with `//sections/terms`, putting it in 3.1.1, 3.1.2

The premise that the boilerplate should be repeated for each instance of a terms collection might be defensible in principle, but it clearly should not apply here. Blocking it from ancestor clauses of terms collections in ISO does not make sense either.

**Comment by opoudjis** [comment]
No. You have explicitly asked for the opposite in https://github.com/metanorma/metanorma-iso/issues/325, and that's why that is implemented.

ISO has made a decision on where the boilerplate should go, and it was actioned. If you want different behaviour, you need to tell me what the revised rule is. I will not implement this without explicit guidance from ISO.

**Comment by ronaldtse** [comment]
Not true, https://github.com/metanorma/metanorma-iso/issues/325#issuecomment-590272264 was never asked for.

The correct behavior here for ISO typical deliverables is that the boilerplate only occurs once in Clause 3, at the highest level where "Terms and definitions" are present. 

## Scenario 1

Clause 3 is "Terms and definitions", 3.X are terms.

=> Boilerplate exists in Clause 3 prior to 3.1.

## Scenario 2

Clause 3 is "Terms and definitions", 3.X are categories, and 3.X.Y are terms.

=> Boilerplate exists in Clause 3 prior to 3.1. Because the highest common clause of all terms is Clause 3.

## Scenario 3

Clause 3 is a combined clause: "Terms, definitions, symbols", "Terms, definitions and abbreviated terms" or "Terms, definitions, symbols and abbreviated terms".

Then Clause 3.1 is "Terms and definitions". Terms are in Clause 3.1.X.

=> Boilerplate exists in Clause 3.1 before 3.1.1.

## Scenario 4

Clause 3 is a combined clause: "Terms, definitions, symbols", "Terms, definitions and abbreviated terms" or "Terms, definitions, symbols and abbreviated terms".

Clause 3.1 is "Terms and definitions". 3.1.X are categories, and 3.1.X.Y are terms.

=> Boilerplate exists in 3.1 prior to 3.1.1. Because the highest common clause of all terms is 3.1.

## Scenario 5 (ISO "Vocabulary" document, #473)

Clause 3 and subsequent sections are vocabularies. Then Clause 3 contains the boilerplate. All other terminology clauses (e.g. Clause 4) will need to have a title that starts with "Terms related to".

> A vocabulary is the only ISO document that can have terminological entries in clauses other than Clause 3. If terminological entries are given in other clauses, use a clause title starting “Terms related to”. Terminological entries are never included in annexes.

The Clause 3 boilerplate needs to be modified such that:
> Do not include the first line of the fixed text for Clause 3, i.e. “For the purposes of this document, the following terms and definitions apply.” This is not needed in a vocabulary document because the terminological entries apply to all the documents of the committee.

Terminology clauses outside Clause 3 do not need the terminology boilerplate.

**Comment by opoudjis** [comment]
Introducing here the docsubtype `vocabulary`.

**Comment by ronaldtse** [comment]
Reopening. Not fixed at git master of metanorma-iso/isodoc.

**Comment by ronaldtse** [comment]
@opoudjis says this is fixed in #498, but I cannot verify due to an error.

**Comment by ronaldtse** [comment]
Verified fixed.
